CoEpi/ui/alerts/AlertsViewModel.swift

```diff
@@ -1,3 +1,21 @@
+import RxCocoa
+
 class AlertsViewModel {
+    private(set) var alerts: Driver<[Alert]>
+    
+    init() {
+        let tempAlerts: [Alert] = [
+            Alert(id: "a", exposure: "testa"),
+            Alert(id: "b", exposure: "testb"),
+            Alert(id: "c", exposure: "testc"),
+            Alert(id: "d", exposure: "testd"),
+        ]
+        alerts = BehaviorRelay(value: tempAlerts)
```

Comment:
There's no reason to use `BehaviorRelay` here. To return a fixed value, use `Observable.just(value)`